src/frontend/src/components/Posts/Posts.jsx

```diff
@@ -46,6 +46,7 @@ const Posts = () => {
   const [posts, setPosts] = useState([]);
   const [numPages, setNumPages] = useState(1);
   const [endOfPosts, setEndOfPosts] = useState(false);
+  const { searchPostResults } = props;
```

Comment:
So it makes more sense to pass in the search results here vs. doing it in this component?

I would have thought doing:

```jsx
<Posts search="This is my search text" author="Calvin Ho" />
```

would have been more logical.  But maybe the code doesn't allow this easily?

Reply:
We have two separate components, depending on what type of search is being carried out: if by posts, we reuse the post component and just pass it everything,

If they search by author: we have an author results component that is displayed instead with no posts, so I'm not sure we'll need to pass the author information

Reply:
OK, we can evolve this in follow up PRs then.  Let's keep this one focused on getting results into Posts.